package.json

```
@@ -8,7 +8,7 @@
     "lint": "npx eslint test/ index.js"
   },
   "engines": {
-    "node": "^10.17.0"
+    "node": ">= 10.17.0"
```

Hi @caiohsramos, good catch!

I think this just needs changing to the following to get it working in Node 12 as well.

```suggestion
    "node": "^10.17.0 || >= 11.14.0"
```